.gitignore

```diff
@@ -1,3 +1,4 @@
 phpunit.xml
 composer.lock
 vendor
+.idea
```

Comment:
It is considered best prectise that IDE specific 'ignores' should be in you systems global ignore file and not in a specific project.